### ISSUE TITLE
Nan loss in training specter with specter/scripts/pytorch_lightning_training_script/train.py 

### ISSUE THREAD

**Issue by AlexRossi6** [allenai/specter#26]
By following the instructions given in the readme:
python train.py --save_dir path/to/save --gpus 1 --train_file path/to/training_set --dev_file path/to/val_set --test_file path/to/test_set --batch_size 2 --num_workers 0 --num_epochs 4 --grad_accum 256

I get nan loss and val_loss is constant:

![image](https://user-images.githubusercontent.com/56583448/114665444-557f5800-9cfd-11eb-993f-8e155b8c902e.png)
